pkg/util/converter/jsonitere/jsonitere.go

```diff
@@ -0,0 +1,58 @@
+// package jsonitere wraps json-iterator/go's Iterator methods with error returns
+// so linting can catch unchecked errors.
+package jsonitere
```

Comment:
I think I may end up moving this into our go-plugin-sdk repo since the dataframe parser uses `json-iterator/go` as well. But it will be easy enough to remove this and change any imports if/when that happens.